server/channels/store/sqlstore/user_store.go

```diff
@@ -1674,7 +1674,20 @@ func (us SqlUserStore) performSearch(query sq.SelectBuilder, term string, option
 
 func (us SqlUserStore) AnalyticsGetInactiveUsersCount() (int64, error) {
 	var count int64
-	err := us.GetReplicaX().Get(&count, "SELECT COUNT(Id) FROM Users WHERE DeleteAt > 0")
+	query := us.getQueryBuilder().
+		Select("COUNT(Id)").
+		From("Users").
+		LeftJoin("Bots ON Users.ID = Bots.UserId").
+		Where(sq.And{
+			sq.Gt{"Users.DeleteAt": 0},
+			sq.NotEq{"Bots.UserId": nil},
```

Comment:
The previous query was `SELECT COUNT(Id) FROM Users LEFT JOIN Bots ON Users.ID = Bots.UserId WHERE Users.DeleteAt > 0 AND Bots.UserId IS NULL`. Is it correct to check for `NotEq` here?

Reply:
wow, that was a typo! thanks for catching it!